scripts/linux-aarch64/install.sh

```diff
@@ -11,12 +13,75 @@ read -p "Absolute install directory path (default: $HOME/albyhub): " USER_INSTAL
 
 INSTALL_DIR="${USER_INSTALL_DIR:-$HOME/albyhub}"
 
+verify_package() {
+  local archive_file="${1}"
+  local overridden_manifest_name="${2}"
```

Comment:
the variable name was a bit confusing to be. Could it be `filename_in_manifest`?